src/cast_core/test/StreamStatsReporter_test.cpp

```diff
@@ -56,7 +56,9 @@ TEST_CASE_METHOD(MongoTestFixture, "StreamStatsReporter successfully connects to
         try {
             auto startCommand = bsoncxx::builder::stream::document{}
                 << "streams_startStreamProcessor" << ""
+                << "tenantId" << "test"
                 << "name" << "sp"
+                << "processorId" << "sp"
```

Comment:
I'd like to see this tested with "test_processorId" instead of repeating "sp", since I think it's not catching a bug in the implementation.

Reply:
The `processorId` value here does not matter too much for us, I just need to specify some value here since we plan to make it a required field. But yeah, I can use test_processorId as well. Done.